automation-tests/test/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+    "compilerOptions": {
+        "moduleResolution": "node",
+        "types": [
+            "node",
+            "webdriverio/async",
+            "@wdio/mocha-framework",
+            "expect-webdriverio",
+            "wdio-vscode-service"
```

Comment:
there may be some missing deps in the package.json as when I tried to verify moving this it complained about not finding some webdriverio/async